profile/README.md

```diff
@@ -26,16 +26,16 @@ You can also browse by Awesome Lists created by the community.
 
 3. Now either:
 
-     - **Start a new community extension** by [opening an issue](https://github.com/Camunda-Community-Hub/community/issues/new/choose) using the `New Community Extension Proposal` template and following [this documentation.](https://docs.github.com/en/github/administering-a-repository/transferring-a-repository#transferring-a-repository-owned-by-your-organization)
-     - [**Transfer your existing repository**](community/transferring-extensions.md) into the Camunda Community Hub.
-     - **Browse our existing projects** and issues and contribute.
+     - **Start a new community extension** by [opening an issue](https://github.com/Camunda-Community-Hub/community/issues/new/choose) using the `New Community Extension Proposal` template.
+     - [**Transfer your existing repository**](https://github.com/camunda-community-hub/community/blob/main/transferring-extensions.md) into the Camunda Community Hub.
```

Comment:
Does this link still work if you use `../../../community/transferring-extensions.md` as the link instead?

Reply:
Not in this repo, because that doc is in the camunda-community-hub/community/ and this one is the main repo for the whole community.

Reply:
Now your Slack messages are making sense! Thanks for clarifying.